bpf/Makefile

```diff
@@ -230,6 +230,18 @@ bpf_lxc.o:: bpf_lxc.c $(LIB)
 
 $(foreach OPTS,$(LXC_OPTIONS),$(eval $(call PERMUTATION_template,bpf_lxc.o,$(OPTS))))
 
+bpf_maps.o:: bpf_maps.c $(LIB)
```

Comment:
Is bpf_maps a permanent fixture or something that can be removed later on? In an ideal world, I think we'd be able to just compile and scan through all the existing .o files to build the registry instead of having another place to keep things synchronized.